### ISSUE TITLE
Get an open source certificate for code signing

### ISSUE THREAD

**Issue by DarkWanderer** [DarkWanderer/Elite-Log-Agent#13]
[Certum](https://www.certum.eu/en/cert_offer_en_open_source_cs/) provides those for cheap

**Comment by DarkWanderer** [comment]
Certum does not provide non-physical key certificates anymore

**Comment by DarkWanderer** [comment]
Looks like Certum has created a [new product](https://en.sklep.certum.pl/data-safety/code-signing-certificates/open-source-code-signing-on-simplysign.html) just for that - need to revisit

**Comment by jakaarl** [comment]
Yes, Chrome is strongly in the opinion the installer is fishy/malware, and Windows isn't exactly encouraging installing it either. If fixing this requires a bit of dough, I'm sure thankful users would chip in for the cost - I would.

**Comment by DarkWanderer** [comment]
Does Chrome actually warn of installer as malware? Never seen it, do you have a screenshot?

I plan to get a hardware key soon:tm: then will be able to look into getting builds signed
